File: user-service/src/main/java/org/mounanga/userservice/dto/UserResponse.java
package org.mounanga.userservice.dto;

import lombok.*;

import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@ToString
public class UserResponse {
    private Long id;
    private String email;
    private String username;
    private Boolean enabled;
    private String createBy;
    private LocalDateTime createdDate;
    private String lastModifiedBy;
    private LocalDateTime lastModifiedDate;
    private ProfileResponse profile;
}

File: admin-bank-ui/src/app/dto/security/user.response.ts
export class UserResponse {
  id!: string;
  firstname!: string;
  lastname!: string;
  username!: string;
  email!: string;
  cin!: string;
  enabled!: boolean;
  createdDate!: Date;
  lastModifiedDate!: Date;
  createBy!: string;
  lastModifiedBy!: string;
  roles!: Set<string>;
}
